src/Blocks/BlockCrops.h

```diff
@@ -23,11 +23,35 @@ class cBlockCropsHandler :
 	{
 		cFastRandom rand;
 
-		if (a_Meta == 0x7)
+		if (a_Meta == 0x3)
```

Comment:
NVM it's just the diff beeing funny 😁 

But I wonder if there is a constant for all the states such a block may have.